src/body/stream_body.rs

```diff
@@ -36,7 +36,7 @@ impl<T> StreamBody<T> {
 
 impl<T> Body for StreamBody<T>
 where
-    T: Stream<Item = Result<Frame<Bytes>, DynError>> + Send + Sync,
+    T: Stream<Item = Result<Bytes, DynError>> + Send + Sync,
```

Comment:
I think I'm going to impose an `Unpin` requirement here for API safety reasons. If someone needs to use a `!Unpin` stream as the source for a response body, they can do so by using BoxBody directly with `Response::new` constructor method.